defs/scenarios/storage/ceph/ceph-mon/pg_imbalance.yaml

```diff
@@ -9,6 +9,14 @@ checks:
       property:
         path: hotsos.core.plugins.storage.ceph.CephCluster.osds_pgs_suboptimal
         ops: [[length_hint], [gt, 0]]
+  pool_utilisation_over_limit:
```

Comment:
I think the intent here is to check whether the cluster has any non-empty pools.  The current name begs the question, "What limit?" and the arbitrary 2% value isn't explained anywhere in the code.

What do you think about changing the scenario name to be `cluster_has_non_empty_pools` or maybe `cluster_has_ingested_user_data`?

Reply:
Agree. I have re-named few the vars & the function and commented on the rationale for choosing 2%. Please take another look.